### PR TITLE
Updating image read me with Az module information

### PR DESCRIPTION
**Pull request by niadak**
Updating image read me with Az module information

### CODE REVIEW

File: images/win/Vs2017-Server2016-Readme.md [hunk 1]
```diff
@@ -345,6 +345,12 @@ _Location:_ C:\Modules\azurerm_5.1.1\AzureRM\5.1.1\AzureRM.psd1
 This version is saved but not installed
 _Location:_ C:\Modules\azurerm_6.7.0\AzureRM\6.7.0\AzureRM.psd1
 
+## Az Powershell modules
+
+#### 1.0.0
+
+This version is saved but not installed
+_Location:_ C:\Modules\az_1.0.0\Az\1.0.0\Az.psd1
```
Comment (kmkumaran) [inline images/win/Vs2017-Server2016-Readme.md]: One of the feedback Chris had was that, users need guidance on how to specify the version as part of their pipeline. Can you also add a line _Guideline to use in pipelines_ and provide this link https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-powershell?view=azure-devops

File: images/linux/Ubuntu1604-README.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ The following software is installed on machines in the Hosted Ubuntu 1604 pool
 - Ansible (ansible 2.0.0.2)
 - AzCopy (azcopy 7.3.0-netcore)
 - Azure CLI (azure-cli                         2.0.58)
+- Az Module (available through the [Azure PowerShell](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-powershell?view=azure-devops) task)
```
Comment (kmkumaran) [inline images/linux/Ubuntu1604-README.md]: Do you need to specify about Task here?

File: images/win/Vs2017-Server2016-Readme.md [hunk 3]
```diff
@@ -345,13 +345,17 @@ _Location:_ C:\Modules\azurerm_5.1.1\AzureRM\5.1.1\AzureRM.psd1
 This version is saved but not installed
 _Location:_ C:\Modules\azurerm_6.7.0\AzureRM\6.7.0\AzureRM.psd1
 
+> Note: You can use a spacific Azure PowerShel version in Pipeline using [Azure PowerShell](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-powershell?view=azure-devops) task.
```
Comment (kmkumaran) [inline images/win/Vs2017-Server2016-Readme.md]: Can you run this through with @RoopeshNair

File: images/linux/Ubuntu1604-README.md (hunk 2 above)
Comment (niadak) [reply images/linux/Ubuntu1604-README.md]: This is existing pattern in the Readme files in the repo. For example, 
- Python (available through the [Use Python Version](https://go.microsoft.com/fwlink/?linkid=871498) task)

File: images/win/Vs2017-Server2016-Readme.md (hunk 3 above)
Comment (niadak) [reply images/win/Vs2017-Server2016-Readme.md]: Sure.

File: images/win/Vs2017-Server2016-Readme.md [hunk 4]
```diff
@@ -345,6 +345,16 @@ _Location:_ C:\Modules\azurerm_5.1.1\AzureRM\5.1.1\AzureRM.psd1
...
```
Comment (kmkumaran) [inline images/win/Vs2017-Server2016-Readme.md]: Nit - typo in spelling for "specific". I see this in other places where you have added as well.

**Comment by niadak** [comment]
Created a new PR with the same change.
https://github.com/Microsoft/azure-pipelines-image-generation/pull/876

**Pull request closed without merging**